File: Ruby/1046.rb
inicio, fim = gets.chomp.split().map{|x|x.to_i}
tempo = 0
if (inicio == fim) then
	tempo = 24
elsif (inicio > fim) then
	tempo = 24 - inicio + fim
elsif (inicio < fim) then
	tempo = fim - inicio
end
if (tempo > 24) then
	tempo = 24
end
puts ("O JOGO DUROU %d HORA(S)" % tempo)

File: Python/1046.py
inicio, fim = list(map(int, input().split()))
tempo = 0
if (inicio == fim):
    tempo = 24
elif (inicio > fim):
    tempo = 24 - inicio + fim
elif (inicio < fim):
    tempo = fim - inicio
if (tempo > 24):
    tempo = 24
print(f"O JOGO DUROU {tempo} HORA(S)")

